components/PinCreate/index.tsx

```diff
@@ -19,12 +19,16 @@ import {useHistory} from 'react-router-native'
 
 import * as Keychain from 'react-native-keychain'
 
-import AppHeaderLarge from '../AppHeaderLarge/index.js'
-import LoadingOverlay from '../LoadingOverlay/index.js'
+import AppHeaderLarge from '../AppHeaderLarge/index'
+import LoadingOverlay from '../LoadingOverlay/index'
 
 import AppStyles from '../../assets/styles'
 
-function PinCreate(props) {
+interface IPinCreate {
+  setupScreens: any
```

Comment:
This type can be a number, maybe an integer, but I don't recall the types with use of negative numbers off the top of my head.